src/command_modules/azure-cli-devops/azure/cli/command_modules/devops/custom.py

```diff
@@ -0,0 +1,215 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+import json
+import os
+import re
+import subprocess
+import time
+from subprocess import check_output
+
+import requests
+import yaml
+
+import azure.cli.core._logging as _logging
+from azure.cli.command_modules.devops import git_repo_parser
+from azure.cli.core._profile import Profile
+# pylint: disable=too-few-public-methods,too-many-arguments,no-self-use
+from azure.cli.core._util import CLIError
+
+logger = _logging.get_az_logger(__name__)
+# LOCALRUN change this to "http://localhost:44454"
+BASE_URL = "https://management.azure.com"
+RESOURCE_BASE_URL = "/subscriptions/{subscription_id}/resourceGroups/{resource_group_name}"
+CONTAINER_SERVICE_BASE_URL = RESOURCE_BASE_URL + "/providers/Microsoft.ContainerService"
+CONTAINER_SERVICE_RESOURCE_URL = CONTAINER_SERVICE_BASE_URL + "/containerServices/{container_service_name}"
+VISUAL_STUDIO_BASE_URL = RESOURCE_BASE_URL + "/providers/microsoft.visualstudio"
+VISUAL_STUDIO_ACCOUNT_URL = VISUAL_STUDIO_BASE_URL + "/account/{vsts_account_name}"
+VISUAL_STUDIO_PROJECT_URL = VISUAL_STUDIO_ACCOUNT_URL + "/project/{vsts_project_name}"
+RP_URL = BASE_URL + CONTAINER_SERVICE_RESOURCE_URL + "/providers/Microsoft.Mindaro"
+API_VERSION = "2016-11-01-preview"
+
+def add_release(name, resource_group_name, remote_url, remote_access_token, vsts_account_name, vsts_project_name, registry_resource_group, registry_name):
+    """
+    Adds a deployment to an existing ACS cluster
+
+    :param name: repo name
+    :type name: String
+    :param resource_group_name: Resource group name
+    :type resource_group_name: String
+    :param remote_url: Repository url
+    :type remote_url: String
+    :param remote_access_token: Access token
+    :type remote_access_token: String
+    :param vsts_account_name: VSTS account name
+    :type vsts_account_name: String
+    :param vsts_project_name: VSTS project name
+    :type vsts_project_name: String
+    :param registry_resource_group: Registry esource group name
+    :type registry_resource_group: String
+    :param registry_name: Registry name
+    :type registry_name: String
+    """
+    # Ensure docker-compose file is correct.
+    ensure_docker_compose()
+
+    #Call the RP
+    call_rp_configure_cicd(name, resource_group_name, vsts_account_name, vsts_project_name, registry_name, registry_resource_group, get_valid_remote_url(remote_url, remote_access_token), remote_access_token)
+    return
+
+def get_valid_remote_url(remote_url = None, remote_access_token = None):
```

Comment:
These internal methods should start with underscore as they aren't exposed as a handler for a command.